holoviews/core/data.py

```diff
@@ -714,7 +717,6 @@ def reshape(cls, eltype, data, kdims, vdims):
         kdim_param = element_params['kdims']
```

Comment:
We should probably renamed ``reshape`` to ``init`` as it is confusing. I got confused with reshape in numpy!